wasm.md

```diff
@@ -254,7 +254,8 @@ Note that we do not need to call `#chop` on the results here.
 ```k
     syntax IBinOp ::= "div_u" | "rem_u"
  // -----------------------------------
-    rule <k> ITYPE . div_u I1 I2 => #if I2 =/=Int 0 #then < ITYPE > (I1 /Int I2) #else undefined #fi ... </k>
+    rule <k> ITYPE . div_u I1 I2 => < ITYPE > (I1 /Int I2)  ... </k> requires I2 =/=Int 0
+    rule <k> ITYPE . div_u I1 I2 => undefined ... </k>               requires I2  ==Int 0
```

Comment:
we should make a consistent choice on whether to branch on these instructions, in which case  `rem_{u,s}` and `div_s` should be updated as well

Reply:
Branching (as this does here) is probably better.

Reply:
At least it give a lot nicer behavior tree in KLab, which is why I changed it for `div_u` (it's the only one I used in the presentation). I'll change it for the others as well.